R/zzz.R

```diff
@@ -16,3 +16,21 @@ bcmaps_env <- new.env(parent = emptyenv())
   assign("bcmaps_update_message", FALSE, envir = bcmaps_env)
 }
 
+
+bcmapsStartupMessage <- function() {
+  msg <- "Support for Spatial objects (`sp`) was deprecated in {bcmaps} v1.2.0, and will be removed altogether in the Summer 2023. Please use `sf` objects with {bcmaps}."
```

Comment:
I think we maybe try to be less precise with this estimate and just say something like "a future release"

Reply:
I think we want to be a little informative re: how quickly we want users to adapt, given that we are planning a reasonably short deprecation window?